UI/src/app/components/panel/panel-table/panel-table.component.html

```diff
@@ -1,21 +1,6 @@
 <table #tableComponent id="{{ table.name }}" mat-table [dataSource]="datasource">
```

Comment:
please refactor this part of old code to [id]="table.name"

Reply:
done